thetis/callback.py

```diff
@@ -95,12 +98,19 @@ def __init__(self, filename, varnames, array_dim=1, attrs=None,
     def _expand(self, hdf5file):
         """Expands data arrays by 1 entry"""
         # TODO is there an easier way for doing this?
-        for var in self.varnames + ['time']:
+        for var in self.varnames:
             arr = hdf5file[var]
             shape = arr.shape
             arr.resize((shape[0] + 1, shape[1]))
+        if self.include_time:
+            arr = hdf5file['time']
+            shape = arr.shape
+            arr.resize((shape[0] + 1, shape[1]))
```

Comment:
You are duplicating the code here in lines 106-108. Could plausibly refactor differently to avoid duplication